src/css/index.css

```diff
@@ -139,40 +176,67 @@ h3 {
   background: var(--header-color);
   color: black;
   font-family: "Open Sans", Arial, Helvetica, sans-serif;
-  padding: 1em .5em; }
-  .footer div {
-    padding: .5em; }
-  .footer a {
-    color: black;
-    padding: 5px;
-    margin: 2px;
-    transition: .5s; }
-    .footer a:hover, .footer a:focus {
-      background: white;
-      color: black;
-      border-radius: 10px; }
+  padding: 1em .5em;
+}
+
+.footer div {
+  padding: .5em;
+}
+
+.footer a {
+  color: black;
+  padding: 5px;
+  margin: 2px;
+  transition: .5s;
+}
+
+.footer a:hover, .footer a:focus {
+  background: white;
+  color: black;
+  border-radius: 10px;
+}
+
+.footer ul, .footer li {
+  float: left;
```

Comment:
After all your awesome grid work @PepperAddict?! 😃 
I've been experimenting with flexbox over floats lately. Flex is great for lists like this, and you don't have to worry about fallbacks, as it's supported more widely than grid.
But overall, this is just my personal suggestion. Floats are not an issue, in and of themselves.